.github/workflows/tests.yml

```diff
@@ -16,17 +16,15 @@ jobs:
         suite: [unit, visual]
       fail-fast: false
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Unit tests on Chrome
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
       - run: npm ci
       - run: npm run build -- -f
       - name: Run ${{ matrix.target }} ${{ matrix.suite }} headless test
-        uses: GabrielBB/xvfb-action@v1
-        with:
-          run: npm run test -- -c ${{ matrix.target }} -s ${{ matrix.suite }}
+        run: xvfb-run -a npm test -- -c ${{ matrix.target }} -s ${{ matrix.suite }}
```

Comment:
I think we need `xvfb-run` for firefox to have webGL
https://manpages.ubuntu.com/manpages/kinetic/en/man1/xvfb-run.1.html
https://bugzilla.mozilla.org/show_bug.cgi?id=1375585#c27